backend/config/default.cjs

```diff
@@ -212,10 +212,10 @@ module.exports = {
         image: 'seldonio/seldon-core-s2i-python37:1.10.0',
       },
     ],
-    maxModelSizeGB: 50
+    maxModelSizeGB: 50,
   },
 
   experimental: {
-    v2: false,
+    v2: true,
```

Comment:
We don't want to enable v2 by default, consider adding this in a `local.cjs` file that isn't committed to git.